repose-aggregator/core/core-lib/pom.xml

```diff
@@ -243,26 +237,22 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-core</artifactId>
-            <version>${fasterxml.jackson.version}</version>
         </dependency>
 
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>${fasterxml.jackson.version}</version>
         </dependency>
 
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>${fasterxml.jackson.version}</version>
         </dependency>
         <!-- END OF json Support for log4j2 configuration files -->
 
         <dependency>
             <groupId>com.typesafe</groupId>
             <artifactId>config</artifactId>
-            <version>1.0.2</version>
```

Comment:
`com.typesafe.config` is being used in 
 - _profile-support_
    - _core-support_
       - **core-lib**
       - **valve**
       - **web-application**
    - _services-pom_
       - _service-client-support_
          - **service-client-akka**

It was originally being defined in each of those bottom levels. Since it was not being dependency managed from a common parent, multiple versions were being brought in.